collect_app/src/main/java/org/odk/collect/android/widgets/SelectMultiWidget.java

```diff
@@ -132,6 +138,15 @@ public void onCheckedChanged(CompoundButton buttonView, boolean isChecked) {
                         buttonView.setChecked(true);
                     }
                 }
+
+                // show warning when selected choice value has spaces
+                int index = (int) checkBox.getTag();
+                String value = items.get(index).getValue();
+                if (isChecked && value != null && value.contains(" ")) {
+                    String errorMsg = context.getString(R.string.invalid_space_in_answer, value);
+                    Timber.e(errorMsg);
```

Comment:
This doesn't need to be logged. Logging at level `e` means it gets sent to remote crash logging and it's not an actionable error.

Reply:
ok